### ISSUE TITLE
Suggestion: comparison with clojure-lsp

### ISSUE THREAD

**Issue by ericdallo** [pedrorgirardi/Pep#32]
Hello, congrats for the project, I realized it's pretty similar to [clojure-lsp](https://clojure-lsp.io) which works for sublime text as well and uses clj-kondo analysis to provide most features, maybe you should mention somewhere in the docs the difference between them and why someone should choose one before the other?

Thank you!

**Comment by borkdude** [comment]
I was happy to discover this project. I'm also interested in the trade-offs. Perhaps a bit of communication can save a lot of overlapping development since the features in clojure-lsp are pretty similar and already working for most editors.

**Comment by pedrorgirardi** [comment]
Hi @ericdallo @borkdude ,

It was almost an accident that Pep came to be. I use [Tutkain](https://github.com/eerohele/tutkain), and I only wanted to add "local jump," so I looked at clj-kondo analysis data. The next thing I knew, I was adding commands to navigate to vars, and when I needed something, I would try to implement it using clj-kondo's analysis data.

Working with clj-kondo's analysis data has been a very satisfactory experience!

Sublime LSP is a very active project, and it really does everything one would expect from an LSP client, but one of the motivations for me to switch to Sublime Text it was the minimal setup to work with Clojure(Script). I really appreciate Tutkain's simplicity and doing very little.

At a certain point, I decided to turn Pep into a real option for Clojure programmers who would like to Sublime Text, but I set the constraint of using only built-in Sublime Text and Python modules. I want to keep it small and don't do too much.

clj-kondo has been working very well for Pep, but I'm aware that I'm redoing many things that clojure-lsp already does. clojure-lsp would be a better "engine" for Pep in the long term, and I have been pondering about it.

I'm finishing some work on the data structures for the indexes Pep uses, and I would like to have a version of Pep that it's functional, but after that, I might try to spend some time learning more about clojure-lsp and the LSP protocol in general.

If I take the clojure-lsp path, I would like to set the same constraint of using only built-in modules and implement an LSP client for Pep instead of using Sublime LSP.

Thank you for checking the project and creating this issue. I would love to get your feedback on this. ☺️
